brambling/forms/organizer.py

```diff
@@ -265,6 +266,30 @@ def _post_clean(self):
             self._update_errors(e)
 
 
+class CustomFormForm(forms.ModelForm):
+    class Meta:
+        model = CustomForm
+        exclude = ('event',)
```

Comment:
This particular line will stay the same, though, for now. This issue is resolved by the addition of help text.